symbols/ram.yml

```diff
@@ -549,18 +606,24 @@ ram:
       address:
         EU: 0x23259F4
         NA: 0x2324EB4
+        JP: 0x2326414
       length:
-        EU: 0x14
-        NA: 0x14
+        EU: 0x18
+        NA: 0x18
+        JP: 0x18
       description: |-
         Used to notify scripts waiting for a certain lock to unlock so they can resume their execution.
         
-        1 byte per lock. Exact size isn't confirmed, it could potentially be longer.
+        1 byte per lock.
     - name: GROUND_MEMORY_ARENA_1
       address:
+        EU: 0x2325B00
         NA: 0x2324FC0
+        JP: 0x2326520
       length:
-        NA: 0x1C
+        EU: 0x64000
```

Comment:
This is wrong, the old length was correct

Reply:
The description said "this arena occupies 0x64000 bytes of space". Is the 0x1C only for the metadata?

Reply:
As is stated in the symbol description, see `struct mem_arena`

Reply:
IMO between the symbol name and description, it's easy to confuse this symbol as representing the whole memory arena rather than just the arena metadata. It'd be more clear if the symbol and struct were named `MEMORY_ARENA_1_METADATA` and `mem_arena_metadata`. But I'm not that invested in this symbol, so I'm fine leaving it as is.